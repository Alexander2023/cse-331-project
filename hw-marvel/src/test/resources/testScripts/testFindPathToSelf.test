# Tests that path finding works properly when asked to find the path from a node to itself

CreateGraph g

AddNode g c1

FindPath g c1 c1