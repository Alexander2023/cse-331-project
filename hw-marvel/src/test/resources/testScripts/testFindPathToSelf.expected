# Tests that path finding works properly when asked to find the path from a node to itself

created graph g

added node c1 to g

path from c1 to c1: